pom.xml

```diff
@@ -29,7 +29,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.532.2</version>
+    <version>1.609.1</version>
```

Comment:
@stephenc The changes of this PR requires `workflow-plugin-1.9-beta-2-SNAPSHOT` and this `SNAPSHOT` requieres Jenkins `1.609.1`.